addons/website_sale_stock/models/sale_order_line.py

```diff
@@ -6,11 +6,11 @@
 class SaleOrderLine(models.Model):
     _inherit = 'sale.order.line'
 
-    def _set_shop_warning_stock(self, desired_qty, new_qty):
+    def _set_shop_warning_stock(self, desired_qty, new_qty, product_name='product'):
         self.ensure_one()
         self.shop_warning = _(
-            'You ask for %(desired_qty)s products but only %(new_qty)s is available',
-            desired_qty=desired_qty, new_qty=new_qty
+            'You ask for %(desired_qty)s %(product_name)s(s) but only %(new_qty)s is available',
```

Comment:
this new translation needs to be exported (unless it already exists)

Reply:
it is not i will do it. thanks